content/working_with/manager/update-deployment.md

```diff
@@ -527,19 +527,3 @@ description: old_description
 # deployment update blueprint
 description: new_description
 ```
-
-## Known Issues
```

Comment:
This maybe needs to hang around as it's for a types.yaml, and one from 3.x could still be in use.